src/share/classes/com/sun/btrace/agent/Main.java

```diff
@@ -290,11 +290,27 @@ private static void parseArgs() {
                 fileRollMilliseconds = null;
             }
             if (fileRollMilliseconds != null) {
+                debugPrint("fileRollMilliseconds is " + settings.isUnsafe());
```

Comment:
I think there is a typo here - printing the value of 'unsafe' for 'fileRollMilliseconds'